tests/validator20/get_collapsed_properties_map_test.py

```diff
@@ -0,0 +1,50 @@
+import functools
+from tests.validator20.conftest import get_spec_json_and_url
+from swagger_spec_validator.validator20 import validate_json, deref
+from swagger_spec_validator.validator20 import get_collapsed_properties_type_mapping
+
+
+def get_deref(spec_dict):
+    swagger_resolver = validate_json(
+        spec_dict,
+        'schemas/v2.0/schema.json',
+    )
+    return functools.partial(deref, resolver=swagger_resolver)
+
+
+def test_get_collapsed_properties_type_mapping_simple_case():
+    file_path = '../../tests/data/v2.0/test_polymorphic_specs/swagger.json'
+    swagger_dict, _ = get_spec_json_and_url(file_path)
+
+    swagger_dict['definitions']['GenericPet']['discriminator'] = 'type'
```

Comment:
It was a copy paste leftover ... this test has nothing to share with ``discriminator`` since it is checking only required and not required fields. I'm removing it!